_overviews/core/string-interpolation.md

```diff
@@ -67,14 +67,36 @@ interpolator, all variable references should be followed by a `printf`-style for
 The `f` interpolator is typesafe.  If you try to pass a format string that only works for integers but pass a double, the compiler will issue an
 error.  For example:
 
-    val height: Double = 1.9d
+{% tabs cacheF class=tabs-scala-version %}
```

Comment:
reusing the `cacheF` (not important but maybe uninformative to a reader of source code)